resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
```
curl http://localhost:8080/users?offset=10&limit=10
curl http://localhost:8080/users?page=2&limit=10
```
Could you please elaborate on why these are not equivalent? I have used the first option here to implement pagination in other APIs. As for the backend, why does it need to care weather page or offset was specified by the user, if it is to return the same results anyway?